bigbluebutton-html5/imports/ui/components/presentation/component.jsx

```diff
@@ -237,11 +240,14 @@ class Presentation extends PureComponent {
   }
 
   componentWillUnmount() {
+    this.setState((state) => ({ ...state, isUnmounted: true }));
```

Comment:
Seems like calling `setState` here will have no effect because the component will be destroyed next.

Reply:
Instead of using setState, consider using Session package and setting a variable on it.

`Session.set('presentationComponentUnmounted', true);`